.github/workflows/check-release.yaml

```diff
@@ -0,0 +1,18 @@
+on: push
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
```

Comment:
`check` would be a suitable name here. This is just the job name and not the workflow name.